### DISCUSSION TITLE
主页的布局问题

### DISCUSSION THREAD

**Discussion by Je5s1e** [ppoffice/hexo-theme-icarus#1140] [Q&A]
作者你好，我很喜欢icarus的布局与风格，但是主页显示每篇文章都是全部显示而不是只显示封面+标题+概述，这样使得主页极其冗长，可以参考一下fluid的主页那种显示文章的布局。希望作者可以参考一下

**Comment by ppoffice** [comment]
@God-Chao https://hexo.io/zh-cn/docs/tag-plugins.html#%E6%96%87%E7%AB%A0%E6%91%98%E8%A6%81%E5%92%8C%E6%88%AA%E6%96%AD

**Reply by Je5s1e** [reply]
谢谢作者百忙之中解决我的疑问，问题已解决^_^
